lesson_01/RandyCastro/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Randy's README</title>
+<link rel="stylesheet" href="stylesheet.css">
+</head>
+<body>
+
+<h1>Hello my name is Randy💫</h1>
+
+<p>Hello, I'm Randy Castro, and for the past six years, I've been deeply immersed in the dynamic world of retail management, specifically at Banana Republic. However I am now looking to transition into the tech industry. My ultimate goal is to keep on learning new things and branch out to multiple streams of income and retire early.</p>
+
+<h2>What's the best way to reach me? 📞</h2>
+
+<ol>
```

Comment:
List indent is too far from the left margin, please adjust. Same for the list below.